### DISCUSSION TITLE
Support JSY-MK-194T

### DISCUSSION THREAD

**Discussion by FabriceDONDEYNE** [xlyric/pv-router-esp32#52] [Ideas]
Bonjour,
est-il envisagé une prise en charge de la mesure d'énergie au moyen d'un module JSY-MK194T ?

**Comment by xlyric** [comment]
Bonjour, 
non, ça n'est pas prévu, pour les mesures il y a déjà soit la SCT, soit directement avec un shelly ( EM ou autre ) 
( ou un topic MQTT )
